sources/packages/backend/apps/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -108,6 +108,7 @@ export class StudentAESTController extends BaseController {
   ): Promise<StudentFileDetailsAPIOutDTO[]> {
     return this.studentControllerService.getStudentUploadedFiles(studentId, {
       extendedDetails: true,
+      ministryView: true,
```

Comment:
Can we call the option as `auditUserDetails` rather? as `minitryView` sounds UI oriented IMO.